im/pinyin/pinyin.cpp

```diff
@@ -699,6 +699,9 @@ PinyinEngine::PinyinEngine(Instance *instance)
     KeySym syms[] = {
         FcitxKey_1, FcitxKey_2, FcitxKey_3, FcitxKey_4, FcitxKey_5,
         FcitxKey_6, FcitxKey_7, FcitxKey_8, FcitxKey_9, FcitxKey_0,
+        FcitxKey_KP_1, FcitxKey_KP_2, FcitxKey_KP_3, FcitxKey_KP_4,
+        FcitxKey_KP_5, FcitxKey_KP_6, FcitxKey_KP_7, FcitxKey_KP_8,
+        FcitxKey_KP_9, FcitxKey_KP_0,
```

Comment:
Suggest having a numpadSelectionKeys vector.

The selectionKeys_ is not only used here, so it could be problematic if CommonCandidateList changed its behavior.